libs/src/utils.js

```diff
@@ -104,7 +105,9 @@ class Utils {
   }
 
   static async configureWeb3 (web3Provider, chainNetworkId, requiresAccount = true) {
-    const web3Instance = new Web3(web3Provider)
+    // initializing web3 with a HttpProvider wrapper for multiple providers
+    // ref: https://github.com/trufflesuite/provider-engine
```

Comment:
can you clarify which part of linked readme this code relates to? also per comment there, seems like we should prob reference the pkg thats currently in use

> Originally created for MetaMask, but has been superceded by json-rpc-engine in combination with our eth-json-rpc-middleware. This module is not very actively maintained, so we recommend using that one instead.

Reply:
I updated this comment to directly point to the code where we see that Web3 consumes a provider.